src/Symfony/Component/Profiler/composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "symfony/profiler",
+    "type": "library",
+    "description": "Symfony Profiler Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "http://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.9",
+        "psr/log": "~1.0"
```

Comment:
for now, http-kernel is an hard requirement, you need also the stop-watch component for the TimeDataCollector (can be a soft dependency)